package/scripts/prmon_compress_output.py

```diff
@@ -0,0 +1,159 @@
+#! /usr/bin/env python3
+"""prmon output smart compression script"""
+
+import argparse
+import os
+import sys
+
+try:
+    import pandas as pd
+except ImportError:
+    print("{0: <8}:: This script needs pandas.".format("ERROR"))
+    sys.exit(-1)
+
+
+MEMORY_IO_NETWORK_GPU_USAGE = [
+    "vmem",
+    "pss",
+    "rss",
+    "swap",
+    "rchar",
+    "wchar",
+    "read_bytes",
+    "write_bytes",
+    "rx_packets",
+    "tx_packets",
+    "rx_bytes",
+    "tx_bytes",
+    "gpufbmem",
+    "gpumempct",
+    "gpusmpct",
+]
```

Comment:
I noticed that we always drop the CPU measurements, namely `utime` and `stime`. I think we should also keep those.

Reply:
Sure, we were dropping them in our testing back then but I agree they are useful for in-depth analyses.
Will add.